samples/DataImporter/GraphEngine.DataImporter/TSLCompiler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.CodeDom.Compiler;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Trinity.TSL;
+
+namespace GraphEngine.DataImporter
+{
+    class TSLCompiler
+    {
+        public string Compile(string fpath)
+        {
+            string exePath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+            Process process = new Process();
+            process.StartInfo = new ProcessStartInfo("MSBuild.exe", Path.Combine(exePath, "TSLCompiler.csproj") + " /p:TSLRoot=" + fpath);
```

Comment:
we should also consider cross-platform issues here. For .NET core projects we should use the command "dotnet build".